server/sample.env

```diff
@@ -1,3 +1,3 @@
-JWT_SECRET=
-MONGO_URI=
+JWT_SECRET= secret
+MONGO_URI = "mongodb+srv://alex:zlLn4lMKeGGWsrIn@cluster0.klg3x.mongodb.net/kaban?retryWrites=true&w=majority"
```

Comment:
Please remove this and in future be careful of what we are posting on a public repository and in a private one as well.

Reply:
Okay! for sure, for got about that.